### ISSUE TITLE
BUG. PHQL error while only inserting to table with 2 columns

### ISSUE THREAD

**Issue by etilley** [phalcon/cphalcon#2019]
I have a table in my database that has 2 columns in it. First column is the id and the second one is the name. if i try to run   
```php
$this->modelsManager->executeQuery('INSERT INTO ModelName (column_name) VALUES (\'test\')');
```
I will get "Unknown expression" but if I do 
```php
$this->modelsManager->executeQuery('INSERT INTO ModelName (id, column_name) VALUES ( \'\'  \'test\')');
```
it will add the row. I added an extra column to the table for test and  it would add the row without me specifying the id column.

**Comment by Green-Cat** [comment]
@andresgutierrez fixed in #2328

**Comment by maxgalbu** [comment]
It's probably not fixed on 2.0, but I didn't check
